docs/details/query-analytics.md

```diff
@@ -0,0 +1,91 @@
+# Query analytics under the hood
+
+## Components
+1. [Filters Panel](../get-started/./query-analytics.md#filters-panel)
+2. [Overview Panel](../get-started./query-analytics.md#overview-panel)
+3. [Details Panel](../get-started./query-analytics.md#details-panel)
+
+## How QAN collects data?
+- data are collected every minute (at 0 seconds, like 8:15:00, next in 8:16:00 etc)
+- collected data are represented by buckets
+
+## What is a bucket/How is it created?
+- buckets contains all data captured during one minute interval
+- once a bucket is created it is sent to PMM Server where it is parsed and saved in the clickhouse database. All QAN related data is stored there. Clickhouse is part of PMM Server, or you can use an external one
+- queries in buckets are aggregated by query ID. It means one row in list overview for all queries with same query ID
+- query IDs are calculated different depends on technology and query source 
+    - **MySQL Perfschema** 
+        - query ID is based on DIGEST value from events_statements_summary_by_digest in mysql database 
+        - DIGEST for same query could be different in different versions of MySQL 
+        - DIGEST is generated from query without sensitive data (DIGEST_TEXT), so both queries below will have **same query ID** 
+            ```sh 
+            INSERT INTO people VALUES ('Joe', 'Doe'); 
+            INSERT INTO people VALUES ('John', 'Smith'); 
+            ``` 
+        - with MySQL 8.0 and higher you can use function STATEMENT_DIGEST("your query") to get DIGEST (query ID). See more details on MySQL official website [here](https://dev.mysql.com/doc/refman/8.0/en/encryption-functions.html#function_statement-digest "MySQL Perfschema digest details") 
+    - **MySQL Slowlog** 
+        - query ID is the right-most 16 characters of the MD5 checksum of fingerprint 
+        - fingerprint is query without sensitive data, so like in case of Perfschema both queries below will have **same query ID** 
+            ```sh 
+            INSERT INTO people VALUES ('Joe', 'Doe'); 
+            INSERT INTO people VALUES ('John', 'Smith'); 
+            ``` 
```

Comment:
I think we can move it one level above to not duplicate.
DIGEST_TEXT = fingerprint

Reply:
Moved.